Gems/PhysX/Code/Source/Joint/PhysXJointUtils.cpp

```diff
@@ -209,6 +209,42 @@ namespace PhysX::Utils
         nativeJoint->setPrismaticJointFlag(physx::PxPrismaticJointFlag::eLIMIT_ENABLED, true);
     }
 
+    void InitializePrismaticLimitD6Properties(const JointLimitProperties& properties, physx::PxD6Joint* nativeJoint)
+    {
+
+        if (!nativeJoint)
+        {
+            return;
+        }
```

Comment:
suggestion: As this could fail we might want to return a result from this function (could be `bool` or an `AZ::Outcome` that might be more useful to the caller)

Reply:
I agree, but I've tried to blend it into the codebase here - the same sanity check is other `Initialize****LimitProperties` functions. I think there should assertion in line 214. 
I can add it here and other sibling functions as well. What do you think @hultonha ?

Reply:
Hmm good idea, if this really never should be null an assert is probably the way to go as then something else has gone horribly wrong somewhere and we shouldn't fail gracefully.

Reply:
Added assert in existing code as well.